scripts/fuel_cycle_gui.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
#! /usr/bin/env python
# -*- coding: utf-8 -*-
```

I was seeing import errors on a Mac if this is simply called via `./scripts/fuel_cycle_gui.py`. Let's be safer and point to the first interpreter in the path by default.